WhereInTheWorld/WhereInTheWorld/MapViewController.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  ViewController.swift
+//  WhereInTheWorld
+//
+//  Created by Max Zvyagin on 2/10/20.
+//  Copyright © 2020 Max Zvyagin. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+class MapViewController: UIViewController, PlacesFavoriteDelegate {
+    
+    @IBOutlet var mapView: MKMapView!{
+        didSet{mapView.delegate = self as MKMapViewDelegate}
+    }
+    @IBOutlet var viewLocation: UILabel!
+    @IBOutlet var viewDescription: UILabel!
+    @IBOutlet var favButton: UIButton!
+    
+    
```

Comment:
Why these two new lines? 

You could even organize IBOutlets under a pragma ``MARK: - IBOutlets` and properties under a pragma `MARK: - Properties`.